lib/pbench/server/api/resources/datasets_list.py

```diff
@@ -122,23 +122,23 @@ def _get(
         # Build a SQLAlchemy Query object expressing all of our constraints
         query = Database.db_session.query(Dataset)
         if "start" in json and "end" in json:
-            self.logger.info("Adding start / end query")
+            current_app.logger.info("Adding start / end query")
```

Comment:
Not directly relevant to your PR; but I put these `info` logs in for debugging, and forgot to remove them ... but I don't think we really want them in the final code. If you touch this again and feel like deleting some code, feel free to delete all four of these "Adding ... query" logs!

Reply:
Done, and I fixed up the doc strings.